watertap/flowsheets/full_water_resource_recovery_facility/BSM2_P_extension.py

```diff
@@ -537,6 +537,10 @@ def scale_variables(m):
             if "conc_mass_comp" in var.name:
                 iscale.set_scaling_factor(var, 1e1)
 
+    for unit in ("R1", "R2", "R3", "R4"):
+        block = getattr(m.fs, unit)
+        iscale.set_scaling_factor(block.hydraulic_retention_time, 1e-3)
```

Comment:
Since we are fixing volume, does it make sense to scale this by inverse of volume/flowrate, perhaps using plant influent flowrate as a start?

Reply:
I've just tried doing this, but none of these new scaling factors work

Reply:
what if we don't provide the scaling facotrs for the HRT as old version?

Reply:
Same thing - if we don't provide any scaling factors for HRT, the system fails to solve